parachain/common/src/registry.rs

```diff
@@ -0,0 +1,18 @@
+
+use crate::types::AppID;
+
+pub enum AppName {
+    PolkaETH
+}
+
+pub struct Entry {
+    pub symbol: AppName,
+    pub id: AppID,
+}
+
+pub static REGISTRY: &'static [Entry] = &[
+    Entry {
+        symbol: AppName::PolkaETH,
+        id: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
```

Comment:
How will we be determining an application's ID on polkadot? Might be confusing to have `[0, 0, 0...]` as an app ID, maybe we could hash the string "PolkaEth" into a 20-byte hex string so it's more similar to the `common.Address` datatype over on Ethereum?

Open to ideas on this one.

Reply:
I agree, that needs to be improved. I still need to figure out the right types to use here (byte strings vs byte arrays, etc).